pkg/generator/generator.go

```diff
@@ -48,43 +41,43 @@ import (
 						secret-values.yaml.patch
 */
 
-var (
-	multipleDashPattern     = regexp.MustCompile("-{2,}")
-	invalidCharacterPattern = regexp.MustCompile("[^a-z0-9]+")
-)
-
 type Config struct {
 	Fs               Filesystem
 	DecryptTraverser DecryptTraverser
-	ProjectVersion   string
+
+	Installation string
```

Comment:
Specifying `DecryptTraverser` which is specific to installation (vault) but no `Installation` didn't make sense to me.